### PR TITLE
ENH: Before/after and HEOG

### LINKED ISSUE #309: Matrix ops
@larsoner maybe consider commenting what kind of operation ['vo...,von,vn...->v...'](https://github.com/LABSN/mnefun/blob/97dceaba843b250625d855442e8e719bf0d0fca7/mnefun/_stats.py#L118) is.

### PR DESCRIPTION
**Pull request by larsoner**
1. Adds HEOG and VEOG projectors
2. Adds before/after trace overlays
3. Changes some SVG images to PNG -- page loads *much* faster and is 1/3 the size (26 MB vs 76) in testing.

master:

Screenshot from 2020-09-17 14-04-50

PR:

Screenshot from 2020-09-17 14-27-58

cc @NeuroLaunch you wanted the plot on the left I think, @nordme you wanted the HEOG/VEOG. But we should talk about the VEOG, currently it just gives the same result as the blink projectors...

Closes #309

### CODE REVIEW

**Comment by NeuroLaunch** [comment]
Thanks for making those changes and sharing these images, Eric.  The
active-projectors graphs look great.

**Comment by larsoner** [comment]
I'll go ahead and merge this since it has some nice enhancements, @nordme we can fix the vertical one separately based on whatever you've figured out in another PR

File: mnefun/_stats.py [hunk 1]
```diff
@@ -114,8 +114,11 @@ def _ht2_p(mu, sigma, n_ave, mu_other, sigma_other, n_ave_other, use_pinv):
             sinv = np.linalg.inv(sigma)
         except np.linalg.LinAlgError:
             sinv = np.linalg.pinv(sigma)
-    # Compute the T**2 values:
-    T2 = np.einsum('vo...,von,vn...->v...', mu, sinv, mu)
+    # Compute the T**2 values (just left- and right-multiply the cov inverse
+    # sinv by the mean; a is the left 3 oris, b is right 3 oris, which get
+    # collapsed; v is vertices and ... is our time points, which get
+    # preserved):
+    T2 = np.einsum('va...,vab,bn...->v...', mu, sinv, mu)
```
Comment (larsoner) [inline mnefun/_stats.py:121]: @ktavabi better?

File: mnefun/_stats.py [hunk 2]
```diff
@@ -118,7 +118,7 @@ def _ht2_p(mu, sigma, n_ave, mu_other, sigma_other, n_ave_other, use_pinv):
     # sinv by the mean; a is the left 3 oris, b is right 3 oris, which get
     # collapsed; v is vertices and ... is our time points, which get
     # preserved):
-    T2 = np.einsum('va...,vab,bn...->v...', mu, sinv, mu)
+    T2 = np.einsum('va...,vab,vb...->v...', mu, sinv, mu)
     # Then convert T**2 for p (here, 3) variables and n DOF into F:
```
Comment (ktavabi) [inline mnefun/_stats.py:122]: I assume the original _signature_ signifies matrix multiplication? A*B*A Is this still the same notation?

Comment (larsoner) [reply mnefun/_stats.py:122]: For einsum, any dimension not preserved in the output is multiplied and added to collapse it, so a basic matrix multiplication which could be written in einsum notation as `'ab,bc->ac'`. So this is a left and right matrix multiplication (or really a left vector-matrix and then a vector-vector multiplication reducing to a scalar or so)

**Pull request merged**